RecoLocalTracker/SiPixelRecHits/src/PixelCPEGenericBase.cc

```diff
@@ -0,0 +1,237 @@
+#include "RecoLocalTracker/SiPixelRecHits/interface/PixelCPEGenericBase.h"
+#include "Geometry/TrackerGeometryBuilder/interface/RectangularPixelTopology.h"
+
+namespace {
+  constexpr float micronsToCm = 1.0e-4;
+  auto convertDoubleVecToFloatVec = [](std::vector<double> const& iIn) {
+    return std::vector<float>(iIn.begin(), iIn.end());
```

Comment:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-87155f/15037/llvm-analysis/report-ec62b0.html#EndPath

should this be written as a regular function or marked as const to resolve the static analyzer issue?